src/wallet.js

```diff
@@ -1571,7 +1573,10 @@ const wallet = {
    */
   txExists(txData) {
     const data = this.getWalletData();
-    return txData.tx_id in data['historyTransactions'];
+    if (data && data['historyTransactions']) {
```

Comment:
When can `data` or `data.historyTransactions` be undefined? Maybe we should add a test that fails without this fix.

Reply:
It happens before we load the wallet.

Added the test.